### PR TITLE
Standardized Suite Graphical Setup Initialization: Font and Theme Standardization

### PR DESCRIPTION
**Pull request by Windchild292**
This PR creates a single method to initialize the base graphical setups across the suite. The main enhancement is that all fonts will now be loaded into the suite, which will be useful when add the ability to customize fonts for https://github.com/MegaMek/mekhq/issues/3097.

This both requires and is required for https://github.com/MegaMek/megameklab/pull/1024 and https://github.com/MegaMek/mekhq/pull/3107

### CODE REVIEW

**Review by SJuliez — COMMENTED** [review]
I have one minor issue and a question.

File: megamek/src/megamek/client/ui/swing/MegaMekGUI.java [hunk 1]
```diff
@@ -95,47 +93,28 @@ public void start() {
...
-        System.setProperty("com.apple.mrj.application.apple.menu.about.name", "MegaMek");
-
-        // Add additional themes
-        UIManager.installLookAndFeel("Flat Light", "com.formdev.flatlaf.FlatLightLaf");
-        UIManager.installLookAndFeel("Flat IntelliJ", "com.formdev.flatlaf.FlatIntelliJLaf");
-        UIManager.installLookAndFeel("Flat Dark", "com.formdev.flatlaf.FlatDarkLaf");
-        UIManager.installLookAndFeel("Flat Darcula", "com.formdev.flatlaf.FlatDarculaLaf");
-
-        // this should also help to make MegaMek look more system-specific
+        // TODO : Move Theme setup to MegaMek::initializeSuiteSetups as part of implementing it in
```
[the remark this answers is not in the document]
Comment (Windchild292) [reply megamek/src/megamek/client/ui/swing/MegaMekGUI.java:100]: MegaMekLab and MekHQ can't handle them without larger changes, with themes especially being a bit of a pain.

File: megamek/src/megamek/MegaMek.java [hunk 2]
```diff
@@ -239,6 +239,58 @@ public static String getUnderlyingInformation(final String originProject, final
...
+            if (filename.toLowerCase().endsWith(MMConstants.TRUETYPE_FONT)) {
+                try (InputStream fis = new FileInputStream(directory.getPath() + '/' + filename)) {
+                    GraphicsEnvironment.getLocalGraphicsEnvironment().registerFont(
+                            Font.createFont(Font.TRUETYPE_FONT, fis));
+                } catch (Exception ex) {
+                    LogManager.getLogger().error("Failed to parse font", ex);
+                }
+                continue;
+            }
+
```
[the remark this answers is not in the document]
Comment (Windchild292) [reply megamek/src/megamek/MegaMek.java]: That would be better, updated

**Review by SJuliez — APPROVED** [review]
All good.

**Pull request merged**